salt/modules/event.py

```diff
@@ -216,13 +216,13 @@ def send(
         if isinstance(with_grains, list):
             data_dict["grains"] = _dict_subset(with_grains, __grains__)
         else:
-            data_dict["grains"] = __grains__
+            data_dict["grains"] = dict(__grains__)
```

Comment:
@dwoz do we want `.value()` here instead?

Reply:
Yes
```suggestion
            data_dict["grains"] = __grains__.value()
```